src/pack_unpack.c

```diff
@@ -348,6 +348,33 @@ static json_t *pack_string(scanner_t *s, va_list *ap)
     }
 }
 
+static json_t *pack_object_inter(scanner_t *s, va_list *ap, int need_incref)
+{
+    int nullable;
+    json_t *json;
+
+    next_token(s);
+    nullable = token(s) == '?';
+    if (!nullable)
+        prev_token(s);
+
+    json = va_arg(*ap, json_t *);
+
+    if (json) {
```

Comment:
The rest of the code does not use `{}` for single statements conditional branches, we should do the same in this function to be consistent.

Reply:
Fixed. Thanks!